public/room.html

```diff
@@ -120,25 +122,41 @@
                 camera="userHeight: 1.6"
                 personal-space-bubble
                 look-controls
-                networked="template: #head-template; showLocalTemplate: false;" 
+                networked="template: #head-template; showLocalTemplate: false;"
             ></a-entity>
 
             <a-entity
                 id="body"
                 body-controller="eyeNeckOffset: 0 -0.11 0.09; neckHeight: 0.05"
-                networked="template: #body-template;" 
+                networked="template: #body-template;"
             ></a-entity>
-            
+
             <a-entity
                 id="nametag"
                 networked="template: #nametag-template; showLocalTemplate: false;"
             ></a-entity>
 
+                <!-- oculus-touch-controls-extended="
+                     dpad: true;
+                     dpad_deadzone: 0.8;
+                     dpad_livezone: 0.3;
+                     dpad_directions: 8;
+                     dpad_turbo: true;
+                     dpad_haptic_intensity: none" -->
```

Comment:
If this is active on #left-hand or #right-hand while you try to use vive controllers, then it will subscribe to "axismove" thinking it is the oculus touch controller's joystick. So I commented this component out.

Reply:
I think what I actually want to do is add this component at runtime iff an oculus touch controller is found (and likewise for the vive-controls-extended component), but I'm not yet sure how to do this.

Reply:
They're both enabled for now.